scss/components/time.scss

```diff
@@ -11,17 +11,17 @@ $block: #{$fd-namespace}-time;
       max-width: fd-space(10);
       text-align: center;
       margin-right: fd-space(1);
+
       &:last-child {
         margin-right: 0;
       }
     }
 
-    &__input {
-        margin: fd-space(2) 0;
-        input {
-          padding-left: fd-space(2);
-          padding-right: fd-space(2);
-          text-align: center;
-        }
+    // Spacing and narrower side padding for child inputs
+    &__item > &__input {
+      margin: fd-space(2) 0;
+      padding-left: fd-space(2);
+      padding-right: fd-space(2);
+      text-align: center;
```

Comment:
You can leverage the power of sass and write this like:
```scss
&__item {
    {{ existing styles }}

    .#{$block}__input {
        margin: fd-space(2) 0;
        padding-left: fd-space(2);
        padding-right: fd-space(2);
        text-align: center;
    }
}
```
I think it's safer to write `fd-time__input` as a descendant class to `fd-time__item`, but not necessarily a direct descendant (using the `>` notation).

Reply:
Yeah I went back and forth on both of those points :)

Changes made.